dvc/cli.py

```diff
@@ -85,6 +86,42 @@
 ]
 
 
+def _find_cmd_suggestions(cmd_arg, cmd_choices, cmd=None):
+    """Find similar command suggestions for a typed command that contains typos.
+
+    Args:
+        cmd_arg: command argument typed in.
+        cmd_choices: list of valid dvc commands to match against.
+
+    Returns:
+        String with command suggestions to display to the user if any exist.
+    """
+    if cmd:
+        suggestion_str = (
+            f"dvc: '{cmd} {cmd_arg}' is not a dvc command. See 'dvc --help'\n"
+        )
+    else:
+        suggestion_str = (
+            f"dvc: '{cmd_arg}' is not a dvc command. See 'dvc --help'\n"
+        )
```

Comment:
it is strange to have it duplicated like so, just use a base string and add what you need dynamically

Reply:
Updated this 👍